DataFormats/Detectors/Common/include/DetectorsCommonDataFormats/DetID.h

```diff
@@ -78,17 +78,19 @@ class DetID
   static constexpr ID TST = 15;
   static constexpr ID CTP = 16;
   static constexpr ID FOC = 17;
+  static constexpr ID FOP = 18;
+  static constexpr ID FOH = 19;
```

Comment:
We would prefer the naming convention 

FOC -> FOCAL pads
FPI -> FOCAL pixels
FHC -> FOCAL HCAL

SInce both pads and pixels start with "p" from the DetID FOP it would not be clear whether it corresponds to pads or pixels, while FPI should be self-explanatory